CommentAgent.py

```diff
@@ -199,11 +199,14 @@ def init_GPT(self) -> None:
         self.GPT.add_message("system", GPTsettings.SYSTEM_PROMPT)
         self.add_pr_messages()
 
-    def add_custom_instructions(self) -> None:
-        file_contents = self.GPT.get_file_contents("agent-settings/README.md")
+    def get_custom_instructions(self) -> str:
+        """Looks for a custom isntructions file, and adds them to the GPT convo"""
```

Comment:
There seems to be a typo in the docstring. Additionally, consider clarifying the message in the formatted string.
        
```suggestion
        def get_custom_instructions(self) -> str:
            """Retrieves custom instructions for the repository from a markdown file if it exists."""
            file_contents = self.GH.get_file_contents("agent-settings/README.md")
            if file_contents != "Could not retrieve file contents.":
                custom_instructions_prompt = f"Repository-specific reviewer guidelines:
{file_contents}"
                return custom_instructions_prompt
            else:
                return ""
```